lib/membrane/realtimer.ex

```diff
@@ -0,0 +1,80 @@
+defmodule Membrane.Realtimer do
+  @moduledoc """
+  Sends buffers to the output in real time, according to buffers' timestamps.
+
+  If buffers come in slower than realtime, they're sent as they come in.
+  """
+  use Membrane.Filter
+
+  def_input_pad :input, caps: :any, demand_unit: :buffers
+  def_output_pad :output, caps: :any, mode: :push
+
+  @impl true
+  def handle_init(_opts) do
+    {:ok, %{timestamp: 0, tick_actions: []}}
+  end
+
+  @impl true
+  def handle_prepared_to_playing(_ctx, state) do
+    {{:ok, start_timer: {:timer, :no_interval}, demand: {:input, 1}}, state}
+  end
+
+  @impl true
+  def handle_process(:input, buffer, _ctx, state) do
```

Comment:
Just to be sure. In ech `tick_actions` list we will always have only one buffer to convey?

Reply:
yes, but events and caps may also be there